### PR TITLE
Update Less function

### PR DESCRIPTION
**Pull request by Thor-wl**
ref: https://github.com/volcano-sh/volcano/issues/1525
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

File: pkg/scheduler/api/resource_info.go [hunk 1]
```diff
@@ -36,6 +36,13 @@ const (
 	minResource float64 = 0.1
 )
 
+const (
+	// Zero means resource dimension not defined will be treated as zero
+	Zero = "Zero"
+	// Infinity means resource dimension not defined will be treated as infinity
+	Infinity = "Infinity"
+)
+
```
Comment (shinytang6) [inline pkg/scheduler/api/resource_info.go:45]: From my personal preference, I tend to define this as a new type, maybe something like
```
type Dimension string

const (
	DimensionZero Dimension= "Zero"
	DimensionInfinity Dimension = "Infinity"
)
```

Comment (Thor-wl) [reply pkg/scheduler/api/resource_info.go:45]: > From my personal preference, I tend to define this as a new type, maybe something like
> 
> ```
> type Dimension string
> 
> const (
> 	DimensionZero Dimension= "Zero"
> 	DimensionInfinity Dimension = "Infinity"
> )
> ```

That looks great. Let me optimize it.

Comment (shinytang6) [reply pkg/scheduler/api/resource_info.go:45]: DimensionDefaultValue is ok for me

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Review by Tammy-kunyu — COMMENTED** [review]
/lgtm

**Review by shinytang6 — APPROVED** [review]
/lgtm
/approve

**Review by wpeng102 — APPROVED** [review]
/lgtm

**Pull request merged**